bot/ping_command.go

```diff
@@ -1,23 +0,0 @@
-package bot
```

Comment:
Figured I could try dog-fooding this by using `!alias ping "!ping" "!echo pong"`